include/lsst/afw/table/pybind11/functorKey.h

```diff
@@ -0,0 +1,57 @@
+#ifndef AFW_TABLE_PYBIND11_FUNCTORKEY_H_INCLUDED
+#define AFW_TABLE_PYBIND11_FUNCTORKEY_H_INCLUDED
+/* 
+ * LSST Data Management System
+ * Copyright 2008-2016  AURA/LSST.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <pybind11/pybind11.h>
+
+#include "lsst/afw/table/FunctorKey.h"
+
+namespace py = pybind11;
+using namespace py::literals;
+
+namespace lsst {
+namespace afw {
+namespace table {
+namespace pybind11 {
```

Comment:
Our coding standard calls for using a subdirectory named `python` not `pybind11`, and using the same rule for the sub-namespace. I believe this is to avoid conflict with the `pybind11` namespace.